src/components/AddressBook/AddressCard/AddressCard.jsx

```diff
@@ -0,0 +1,18 @@
+const AddressCard = ({ address }) => {
+  return (
+    <>
+      <div className="">
+        <p className="">
+          <span className="">{address.name}</span>
+          <span className="">{address.mobile}</span>
+        </p>
+        <p className="">
+          {(address.address, address.city, address.state)} -{" "}
+          <span className="">{address.pincode}</span>
```

Comment:
can remove empty class names

Reply:
thanks for the suggestion, will remove them during refactor